build/builder.sh

```diff
@@ -8,6 +8,11 @@ function init() {
   docker build --tag="${image}" "$(dirname $0)"
 }
 
+function cockroach_dir() {
+  local dir=$(cd $(dirname $0); pwd)
```

Comment:
If you invoke the script as `build/builder.sh`, `$(dirname build/builder.sh)` will return `build` and `$(dirname build)` will return `.`.